ui/src/main/java/edu/wpi/grip/ui/deployment/FRCAdvancedDeploymentOptionsController.java

```diff
@@ -32,11 +34,13 @@ FRCAdvancedDeploymentOptionsController create(
 
     @Inject
     FRCAdvancedDeploymentOptionsController(DeployedInstanceManager.Factory deployedInstanceManagerFactor,
+                                           Pipeline pipeline,
                                            @Assisted Consumer<DeployedInstanceManager> onDeployCallback,
                                            @Assisted("stdOut") Supplier<OutputStream> stdOut,
                                            @Assisted("stdErr") Supplier<OutputStream> stdErr) {
         super("FRC Advanced", onDeployCallback);
         this.deployedInstanceManagerFactor = deployedInstanceManagerFactor;
+        this.projectAddress = pipeline.getProjectSettings().computeNetworkTablesServer();
```

Comment:
We already stated that we are keeping the accordion.
In case you've forgotten, here's the discussion where we made this decision: https://github.com/WPIRoboticsProjects/GRIP/issues/258